examples/getopt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-getopt)
+
+hunter_add_package(getopt)
+find_package(getopt CONFIG REQUIRED)
+
+add_executable(getopt_example main.cpp)
```

Comment:
I guess your example is designed to be used both on Windows and other platforms (?) In this case it should be:
```cmake
add_executable(getopt_example main.cpp)
if(MSVC)
  hunter_add_package(getopt)
  find_package(getopt CONFIG REQUIRED)
  target_link_libraries(getopt_example getopt::getopt)
endif()
```